### PR TITLE
Add code of conduct and coding guidelines 

### PR DESCRIPTION
**Pull request by junchaoxia**
Try to fix the issue #10 for code of conduct and coding guidelines.

### CODE REVIEW

File: CODING_GUIDELINES.md [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+## Coding Style
+
+In order to keep the code consistent, please read and follow [PEP 8 Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008)
+and [PEP 257 Docstring Conventions](https://www.python.org/dev/peps/pep-0257/) guidelines. [The webpage from RealPyhon](https://realpython.com/documenting-python-code/) is a good guide for documenting Python Code. 
+For automatic checking of style problems, [Pycharm] from Jet Brains (https://www.jetbrains.com/pycharm/) 
+is a good candidate which has the professional version for academic users with free license fee. 
+
+
+##Best Practices
```
Comment (vineetbansal) [inline CODING_GUIDELINES.md]: Space after ## needed for proper rendering

File: CODING_GUIDELINES.md [hunk 2]
```diff
@@ -0,0 +1,35 @@
...
+
+For the beginners in scientific computing, some general guides are suggested from Grep Wilson's papers as below:
+
+ 1. [Good Enough Practices in Scientific Computing]( https://doi.org/10.1371/journal.pcbi.1005510) 
+ 2. [Best Practices for Scientific Computing]( https://doi.org/10.1371/journal.pbio.1001745) 
+
+
+##Source Code Control
```
Comment (vineetbansal) [inline CODING_GUIDELINES.md]: Space after ## needed for proper rendering

File: CODING_GUIDELINES.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+
+# Coding Guidelines
+
+This document contains the coding guidelines for the ASPIRE Python Project.
+
+## Coding Style
+
+In order to keep the code consistent, please read and follow [PEP 8 Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008)
+and [PEP 257 Docstring Conventions](https://www.python.org/dev/peps/pep-0257/) guidelines. [The webpage from RealPyhon](https://realpython.com/documenting-python-code/) is a good guide for documenting Python Code.
```
Comment (vineetbansal) [inline CODING_GUIDELINES.md]: RealPyhon -> RealPython
The webpage -> This webpage ("from RealPython" shouldn't be in the hyperlink)

File: CODING_GUIDELINES.md [hunk 4]
```diff
@@ -0,0 +1,35 @@
...
+
+In order to keep the code consistent, please read and follow [PEP 8 Style Guide for Python Code](https://www.python.org/dev/peps/pep-0008)
+and [PEP 257 Docstring Conventions](https://www.python.org/dev/peps/pep-0257/) guidelines. [The webpage from RealPyhon](https://realpython.com/documenting-python-code/) is a good guide for documenting Python Code. 
+For automatic checking of style problems, [Pycharm] from Jet Brains (https://www.jetbrains.com/pycharm/) 
+is a good candidate which has the professional version for academic users with free license fee. 
+
+
+##Best Practices
+
+For the beginners in scientific computing, some general guides are suggested from Grep Wilson's papers as below:
```
Comment (vineetbansal) [inline CODING_GUIDELINES.md]: Greg

File: CODING_GUIDELINES.md (hunk 1 above)
Comment (junchaoxia) [reply CODING_GUIDELINES.md]: done.

File: CODING_GUIDELINES.md (hunk 2 above)
Comment (junchaoxia) [reply CODING_GUIDELINES.md]: done.

File: CODING_GUIDELINES.md (hunk 3 above)
Comment (junchaoxia) [reply CODING_GUIDELINES.md]: fixed.

File: CODING_GUIDELINES.md (hunk 4 above)
Comment (junchaoxia) [reply CODING_GUIDELINES.md]: fixed.

**Pull request merged**